features/step_definitions/receive-steps.js

```diff
@@ -16,3 +16,15 @@ Then('I should see {int} addresses', async function (numberOfAddresses) {
   const addressesFound = await getVisibleElementsCountForSelector(this.client, '.WalletReceive_walletAddress', `.generatedAddress-${numberOfAddresses}`);
   expect(addressesFound).to.equal(numberOfAddresses);
 });
+
+Then('They should be ordered by created date descending', async function () {
```

Comment:
@daniloprates please update this test step to take the input values in a table form.